analyses/npl/npl_settings.py

```diff
@@ -0,0 +1,86 @@
+import os
```

Comment:
So great to have this file and keep things cleaner in the notebooks! For my own curiosity, what's the advantage of keeping this as ```.py``` rather than something like a ```config.yml```?

Also I'd vote to refer to these are parameters rather than settings, but not really a big deal.

Reply:
>  For my own curiosity, what's the advantage of keeping this as .py rather than something like a config.yml?

I guess the advantages of a .py file are:
- easier to import (no mucking around with yaml)
- more flexible format, you can add methods or imports as I did
I prefer to use a yaml file when the hierarchical structure becomes more useful, such as when you have multiple parameter files for different countries but with the same parameter names, for example. Hope that makes sense1

> Also I'd vote to refer to these are parameters rather than settings, but not really a big deal.

Good call, changed the name!